Makefile

```diff
@@ -33,3 +33,8 @@ hotrod-run:
 prom-run:
 	echo "Open browser on :9090"
 	docker run --rm --net=host -v ${PWD}/manifests/prometheus-config.yml:/etc/prometheus/prometheus.yml prom/prometheus:latest
+
+.PHONY: grafana-run
+grafana-run:
+	echo "Open browser on :3000"
+	docker run --rm -it --net=host grafana/grafana
```

Comment:
Small nit: A config file and the dashboard.json can be mounted on the container directly!

Ref - https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards

Reply:
I have added a config file for datasource and dashboards. In the tracequality dashboard I hardcoded datasource `Prometheus`. We can change this once we introduce mixins.